cl/recap/models.py

```diff
@@ -24,6 +28,7 @@ class UPLOAD_TYPE:
     APPELLATE_CASE_QUERY_PAGE = 13
     CASE_QUERY_RESULT_PAGE = 14
     APPELLATE_CASE_QUERY_RESULT_PAGE = 15
+    ACMS_DOCKET_JSON = 16
```

Comment:
Why would that be "best"? A PDF is a PDF, we don't do any source-specific parsing of PDFs (do we?), so why would we want a different endpoint because one came from ACMS and one came from District CMECF?

Similarly, we don't have a different upload type for appellate PDFs versus district PDFs versus bankruptcy PDFs.

Reply:
> and attachment pages?

https://github.com/freelawproject/courtlistener/pull/3393 will add the upload type for attachment pages